docs/src/development/troubleshoot.md

```diff
@@ -128,23 +128,31 @@ To send large files, use the `multipart/form-data` header instead:
 $ curl -XPOST 'https://example.com/graphql' --header 'Content-Type: multipart/form-data' -F file=large_file.json
 ```
 
-## Permission error creating a database
+## Databases
+
+For MySQL specific errors, see [troubleshoot MySQL](../configuration/services/mysql/troubleshoot.md).
+
+### Permission error creating a database
 
 If you try to use a user to create a database, you get an error saying `permission denied to create database`.
 The database is created for you
 and can be found in the `path` key of the `$PLATFORM_RELATIONSHIPS` [environment variable](./variables/use-variables.md#use-platformsh-provided-variables).
 
-## Can't write to file system
+## Storage
+
+If you're having trouble with storage, you can [troubleshoot mounts](../configuration/app/troubleshoot-mounts.md) and [disks](../configuration/app/troubleshoot-disks.md).
+
+### Can't write to file system
 
 If you attempt to write to disk outside a `build` hook, you may encounter a `read-only file system` error.
 Except where you define it, the file system is all read-only, with code changes necessary through git.
 This gives you benefits like repeatable deployments, consistent backups, and traceability.
 
-You can write to disk a `build` hook to generate anything you need later.
+You can write to disk a [`build` hook to generate anything you need later](../configuration/app/app-reference.md#writable-directories-during-build).
 Or you can declare writable [mounts](../configuration/app/app-reference.md#mounts#mounts), which are writable even during and after deploy.
```

Comment:
I think this is too much unrelated info inside a link -- it doesn't really describe what the link is about. Maybe something like:
```suggestion
To generate anything you need later, [write to disk during a `build` hook](../configuration/app/app-reference.md#writable-directories-during-build).
Or [declare mounts](../configuration/app/app-reference.md#mounts#mounts),
which are writable even during and after deploy.
```